resources/js/cart.js

```diff
@@ -0,0 +1,66 @@
+function postCartData(type, data) {
+  return fetch('/cart/' + type, {
+    method: 'POST',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify(data)
+  }).then(res => res.json());
+}
+
+const cartController = (function() {
+  function updateCartTotalCount(totalCount) {
+    document.getElementsByClassName('nav__end-cart-count')[0].innerHTML = totalCount + ' items';
```

Comment:
Cache the result of DOM traversal inside a separate variable on load of this class, instead of traversing the DOM again and again upon each cart total update. And use query selector instead.

Reply:
done